test/integration/embedded_pulsar_singularity_job_conf.yml

```diff
@@ -13,6 +13,8 @@ execution:
       runner: pulsar_embed
       singularity_enabled: true
       singularity_required: true
+      # Since tests run in /tmp/ , we apparently need to forbid the default mounting of /tmp
+      singularity_run_extra_arguments: '--no-mount tmp'
```

Comment:
Completely bonkers defaults btw. Who said something about singularity being safer than docker 🤣

Reply:
Should we make that the default ?

Reply:
Good question? Was actually my first idea .. but I had the feeling that I do not understood the actual problem well enough. Just for our reference: The actual problem in the tests was: 

```
FATAL:   container creation failed: mount /tmp/tmpfr6yrqr6/000/1->/tmp/tmpfr6yrqr6/000/1 error: while mounting /tmp/tmpfr6yrqr6/000/1: destination /tmp/tmpfr6yrqr6/000/1 doesn't exist in container
```

The problem seems only to occur when mounting subdirs of `/tmp`, i.e. `-B /tmp/:/tmp/` works and `-B /tmp/tmpei26zwmg/:/tmp/tmpei26zwmg/ ` doesn't.

Reply:
It's totally unsafe to mount /tmp by default, docker doesn't do it.

Reply:
Created an issue for now https://github.com/galaxyproject/galaxy/issues/15673